src/typography/typography.css

```diff
@@ -0,0 +1,58 @@
+/* This file generated from https://google-webfonts-helper.herokuapp.com/fonts/roboto?subsets=latin */
+
+html,
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+/* roboto-regular - latin */
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 400;
+  src: local("Roboto"), local("Roboto-Regular"),
+    url("../../fonts/roboto-v19-latin/roboto-v19-latin-regular.woff2")
+      format("woff2"),
+    /* Chrome 26+, Opera 23+, Firefox 39+ */
+      url("../../fonts/roboto-v19-latin/roboto-v19-latin-regular.woff")
+      format("woff"); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+/* roboto-italic - latin */
+@font-face {
+  font-family: "Roboto";
+  font-style: italic;
+  font-weight: 400;
+  src: local("Roboto Italic"), local("Roboto-Italic"),
+    url("../../fonts/roboto-v19-latin/roboto-v19-latin-italic.woff2")
+      format("woff2"),
+    /* Chrome 26+, Opera 23+, Firefox 39+ */
+      url("../../fonts/roboto-v19-latin/roboto-v19-latin-italic.woff")
+      format("woff"); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+/* roboto-700 - latin */
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 700;
+  src: local("Roboto Bold"), local("Roboto-Bold"),
+    url("../../fonts/roboto-v19-latin/roboto-v19-latin-700.woff2")
+      format("woff2"),
+    /* Chrome 26+, Opera 23+, Firefox 39+ */
+      url("../../fonts/roboto-v19-latin/roboto-v19-latin-700.woff")
+      format("woff"); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
```

Comment:
@joeldenning why not load the fonts from cdn instead? What's the reason behind storing fonts in this repo?

Reply:
So that it works offline. I thought it would be easier just to host the fonts ourself instead of creating a way to either self host or use via cdn, to cover both offline and online use cases.